src/macros.js

```diff
@@ -731,13 +725,8 @@ defineMacro("\\coloneq",
 // \providecommand*\Coloneq{\dblcolon\mathrel{\mkern-1.2mu}\mathrel{-}}
 defineMacro("\\Coloneq",
     "\\mathrel{\\dblcolon\\mathrel{\\mkern-1.2mu}\\mathrel{-}}");
-// \providecommand*\eqqcolon{=\mathrel{\mkern-1.2mu}\vcentcolon}
-defineMacro("\\eqqcolon", "\\mathrel{=\\mathrel{\\mkern-1.2mu}\\vcentcolon}");
 // \providecommand*\Eqqcolon{=\mathrel{\mkern-1.2mu}\dblcolon}
 defineMacro("\\Eqqcolon", "\\mathrel{=\\mathrel{\\mkern-1.2mu}\\dblcolon}");
-// \providecommand*\eqcolon{\mathrel{-}\mathrel{\mkern-1.2mu}\vcentcolon}
-defineMacro("\\eqcolon",
-    "\\mathrel{\\mathrel{-}\\mathrel{\\mkern-1.2mu}\\vcentcolon}");
```

Comment:
We have a macro `\\html@mathml`, which renders the first argument to HTML and the second to MathML. For example, you can do `\\html@mathml{\\mathrel{\\mathrel{-}\\mathrel{\\mkern-1.2mu}\\vcentcolon}}{\\mathrel{\\char`\u2239}}`.

Reply:
Yes, I can use `\html@mathml` to define MathML  as `\mathrel{\u2254}`. But then we also have a macro:

```
defineMacro("\u2254", "\\coloneqq");
```

That looks to me like an infinite loop.  In order to avoid the infinite loop, I put the code into the function section instead. Is there some macro trick that avoids the infinite loop?

Reply:
@ronkok AFAIK, using `\char[backtick]` won't expand the character after it.

Reply:
@ylemkimon Once again, you are quite right.

There is a very small penalty to using `\char`. The MathML we seek for `\coloneqq` is `<mo>≔</mo>`. Using `\char`, the MathML that we get is 
```
<mo>
    <mi mathvariant="normal">≔</mi>
</mo>
```

The second version renders just fine in Firefox. I consider it to be acceptable and I will change this PR to the macro version that you suggest. Thank you for the help.